### PR TITLE
Insert new event types when first seen.

### PR DESCRIPTION
**Pull request by marcospri**
Instead of keeping them in sync with migrations and other methods for testing.

### CODE REVIEW

File: tests/unit/lms/services/event_test.py [hunk 1]
```diff
@@ -60,6 +60,26 @@ def test_insert_event_no_data(self, svc, db_session):
...
+        )
+        # It will create a new type
+        event_type = type_query.one()
+        # Clear the @lru_cache of the private method
+        svc._get_type_pk.cache_clear()  # pylint:disable=protected-access
+        # Insert the event type again
+        svc.insert_event(
+            BaseEvent(request=sentinel.request, type=EventType.Type.CONFIGURED_LAUNCH)
+        )
+        # It will create a new type
```
Comment (jon-betts) [inline tests/unit/lms/services/event_test.py:80]: This comment doesn't seem quite right? Are we checking we get the same one back?

The arrange, act, assert here is a bit unclear

File: tests/unit/lms/services/event_test.py [hunk 2]
```diff
@@ -60,6 +60,26 @@ def test_insert_event_no_data(self, svc, db_session):
...
+        assert event_type == type_query.one()
```
Comment (jon-betts) [inline tests/unit/lms/services/event_test.py:81]: I'm not clear on what running a query twice does? When does a cursor get bound? Only when calling `one()` etc?

**Review by jon-betts — APPROVED** [review]
I think this is sensible and keeps the management of this all in one place.

File: tests/unit/lms/services/event_test.py (hunk 1 above)
Comment (marcospri) [reply tests/unit/lms/services/event_test.py]: Updated the comment and added a bit of white space.

File: tests/unit/lms/services/event_test.py (hunk 2 above)
Comment (marcospri) [reply tests/unit/lms/services/event_test.py:82]: It just runs it twice, no more magic involve.

**Pull request merged**